constants.go

```diff
@@ -13,6 +13,13 @@ const (
 	TRACE   = "TRACE"
 )
 
+// Constants for PostForm
+const (
+	POSTFORM            = "POST"
+	POSTFORMHeaderKey   = "Content-Type"
```

Comment:
Please create a separate constant group for headers, This header key is not specific to the POSTFORM. 

You can add it as a `CONTENT_TYPE` other typical header types are `CONTENT_LENGTH`, `ACCEPT`

We can create a constant such as 
```
const Header = struct {
            ContentType,
}{"Content-Type"}
```

So when we use this it will something like Header.ContentType

Reply:
As far as I can tell you can't have const structs in go. I could make a pseudo constant struct and wrap it in a function but I personally feel something like 

```go
// Constants for headers
const (
	HeaderContentType = "Content-Type"
	HeaderA                  = "A"
	HeaderB                  = "B"
        ...
)
```

would be better.

Reply:
Hi yes you are right there are no const structs,  I was expecting something like 

```
var headers = struct {
 urlEncoded string
}{"url-encoded"}
``` 

This will help us with the logical grouping of all the headers in one place.

Reply:
fixed